server/routes/repos/[owner]/[repo]/readme.ts

```diff
@@ -2,11 +2,33 @@ import { ghMarkdown } from "~/utils/github";
 
 export default cachedEventHandler(
   async (event) => {
-    const repo = `${event.context.params.owner}/${event.context.params.repo}`;
-    const markdown = await $fetch(
-      `https://raw.githubusercontent.com/${repo}/main/README.md`
-    );
-    const html = await ghMarkdown(markdown, repo, "readme");
+    const prefix = "https://raw.githubusercontent.com";
+    const repo = `${prefix}/${event.context.params.owner}/${event.context.params.repo}`;
+    let markdown = await $fetch(`${repo}/main/README.md`);
+    let html = await ghMarkdown(markdown, repo, "readme");
+
+    const pathReplacer = (
+      match: string,
+      link: string,
+      url: string | undefined,
+      url2: string
+    ) => {
```

Comment:
Perf: We could define a normal function outside for this.

Reply:
@pi0  I defined a helper function outside that can be used elsewhere. Let me know if you have any more suggestions! 

Thanks,
CP